### PR TITLE
Created button component

### PR DESCRIPTION
**Pull request by muskanbagrecha**
Please review.

### CODE REVIEW

File: alert/alert-dark.css [hunk 1]
```diff
@@ -1,13 +1,13 @@
 :root{
     --primary-danger-gradient: linear-gradient(90deg, rgba(221, 44, 24, 0.3) 0%, rgba(255, 255, 255, 0.1) 100%);
-    --primary-danger-text: rgb(255, 0, 0);
+    --primary-danger-text: rgb(199, 28, 28);
 
     --primary-success-gradient: linear-gradient(90deg, rgba(46, 204, 113, 0.2) 0%, rgba(255, 255, 255, 0.1) 100%);
-    --primary-success-text: rgb(8, 207, 8);
+    --primary-success-text: rgb(56, 197, 56);
```
Comment (iRohitGaur) [inline alert/alert-dark.css:6]: try to use hex values

File: badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,128 @@
+:root{
+    --color-online: #1b7052;
+    --color-offline: #ff0019;
+    --color-away: #FFA500;
```
Comment (iRohitGaur) [inline badge/badge.css:4]: suggestion: use variable names in uppercase for better visibility

File: badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+}
+
+.badge{
+    position: absolute;
+    border-radius: var(--bradius-full);
+    border: 2px solid var(--color-white);
+    z-index: 1;
```
Comment (iRohitGaur) [inline badge/badge.css:11]: Try to use badge without z-index, if possible

**Review by iRohitGaur — APPROVED** [review]
nice work. Good to merge with changes 👍🏻

**Review by PuneetP16 — APPROVED** [review]
Good fixes. Good to merge

**Pull request merged**